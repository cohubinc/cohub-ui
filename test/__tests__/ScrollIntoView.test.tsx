import React from "react";
import { mount } from "enzyme";

import { ScrollIntoView } from "dist";

describe("ScrollIntoView", () => {
  it("renders", async () => {
    const wrapper = mount(<ScrollIntoView />);

    expect(wrapper).toMatchSnapshot();
  });
});
