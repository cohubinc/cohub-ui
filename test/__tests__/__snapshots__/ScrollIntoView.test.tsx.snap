// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollIntoView renders 1`] = `
<ScrollIntoView
  className=""
  scroll={true}
  scrollOpts={Object {}}
>
  <div
    className=" w-100"
  />
</ScrollIntoView>
`;
